app/src/main/java/com/sun/mangareader01/helper/Constants.kt

```diff
@@ -0,0 +1,3 @@
+package com.sun.mangareader01.helper
+
+const val PARAM_KEYWORD = "keyword"
```

Comment:
```suggestion
const val BUNDLE_SEARCH_KEY = "BUNDLE_SEARCH_KEY"
```
a nghĩ hằng số này chỉ dùng trong search fragment
em nên đặt nó Trong search fragment